manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,6 +39,14 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: PodsUnableToDelete
+          annotations:
+            summary: Some pods are unable to be deleted.
+            description: Some pods have been unable to be deleted for last 24h (since their grace period for termination has been reached).
+          expr: |
+            count by (namespace) (time() - kube_pod_deletion_timestamp > 86400)
```

Comment:
is having this for every pod useful? Maybe aggregation over namespaces would be better in case there is a large number of pods are stuck?

Reply:
Another approach could be to just alert whenever there is at least one pod stuck for more than 24h and provide a query in a runbook to get the list of all the pods that are stuck.

Alert expression would be:
```
time() - min(kube_pod_deletion_timestamp) > 86400)
```

Runbook expression would be:
```
time() - group by (namespace, pod, instance) (kube_pod_deletion_timestamp) > 86400
```